scripts/ipfs/install

```diff
@@ -38,6 +38,16 @@ ipfs config Pubsub.Router gossipsub
 # Enable Filestore for --nocopy capability
 ipfs config --bool Experimental.FilestoreEnabled true
 
+# Setup connection management - Reduce connections to stress the Pi less
+# XXX: These values need to be tweaked and tested
```

Comment:
@makeworld-the-better-one How will you measure whether these values provide an improvement or otherwise?

Reply:
CPU and RAM usage can be looked, as well as file retrieval times and IPNS resolve. It will require a formal test, but I know that these values will be better than the ones we have now, especially since this is on such limited hardware.